### ISSUE TITLE
Panic when using `Capability::Commands` without `Capability::Tags`

### ISSUE THREAD

**Issue by Chronophylos** [museun/twitchchat#188]
```
panicked at 'msg identity conversion should be upheld: ExpectedTag { name: "user-id" }'
```
Panic occurs at:
https://github.com/museun/twitchchat/blob/227e7be4c02541a2dcac83550e84dc5f076224f1/src/runner/async_runner.rs#L303-L305

**Comment by museun** [comment]
What was the input message for this? Was it during normal operation?

**Comment by Chronophylos** [comment]
This happens when I join a channel

**Comment by museun** [comment]
I fixed this in v0.12.2
